Vagrantfile

```diff
@@ -34,25 +34,23 @@ Vagrant.configure(2) do |config|
     config.vm.box = "ubuntu/vivid64"
     ubuntu_common config
   end
-  config.vm.define "wheezy" do |config|
-    config.vm.box = "debian/wheezy64"
-    deb_common(config)
-  end
+  # Wheezy's backports don't contain Openjdk 8 and the backflips required to
+  # get the sun jdk on there just aren't worth it. We have jessie for testing
+  # debian and it works fine.
```

Comment:
I think it makes sense to add it back once there is proper java 8 support for it by a backporting repo and this release is released and not just master for development... If we dont have java 8 on the VM, we should ignore for now, thus +1000 for the comment